src/Ekreative/UuidExtraBundle/Serializer/UuidNormalizer.php

```diff
@@ -23,7 +23,7 @@ public function denormalize($data, $class, $format = null, array $context = [])
 
     public function supportsDenormalization($data, $type, $format = null)
     {
-        return $type === UuidInterface::class || $type === Uuid::class;
+        return $type === UuidInterface::class || $type === Uuid::class || $type === "Ramsey\Uuid\Lazy\LazyUuidFromString";
```

Comment:
you can use a `::class` here, its not a problem if the class doesnt exist.

Reply:
This would break for users that still run `ramsey/uuid<4.1`.

Reply:
Learned something : PHP may be the single langage where importing a missing class doesn't fail :p.

Tried unit tests with `ramsey/uuid:^3`, it works too.